data/core/titleview.lua

```diff
@@ -96,8 +106,11 @@ end
 
 function TitleView:draw_window_controls()
   for item, x, y, w, h in self:each_control_item() do
-    local color = item == self.hovered_item and style.text or style.dim
-    common.draw_text(style.icon_font, color, item.symbol, nil, x, y, 0, h)
+    -- draw an accented background for a hovered item
+    if item  == self.hovered_item then
+      renderer.draw_rect(x, y, w, h, style.dim)
+    end
+    common.draw_text(style.icon_font, style.text, item.symbol, "center", x, y, w, h)
```

Comment:
Tbh it looked a bit better with the old `local color = item == self.hovered_item and style.text or style.dim` for the icon color.

Reply:
That's because the buttons are overly massive. For instance, when using a more minimal icon font (like codicons):
![image](https://github.com/lite-xl/lite-xl/assets/20792268/c54b1670-7d3a-4b66-aa87-89a01b5d42ee)

It looks amazing.

Reply:
Awesome!